build-examples.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
On Tue, 2015-11-03 at 16:52 -0800, sergiusens wrote:
> In build-examples.sh:
> > @@ -0,0 +1,12 @@
> > +#!/bin/bash
> maybe ./runtests plainbox examples instead?
Yes, didn't know about that when I wrote this. Removing.